### ISSUE TITLE
标注问题

### ISSUE THREAD

**Issue by delicate00** [doccano/doccano#2343]
当我标注的时候，选择文本内容，并未自动弹出标签，这是为什么呢？
![image](https://github.com/doccano/doccano/assets/110662274/024cdcbc-4592-4417-b2cb-c16627f96502)

**Comment by delicate00** [comment]
> Would you write your environment? Thank you!

Operating System: windows10
Python Version Used: Python3.10
doccano==1.8.4
其他环境如下：
--------------------------------------- ------------
absl-py                                 2.1.0
accelerate                              0.27.2
affinegap                               1.12
aiohttp                                 3.9.3
aiosignal                               1.3.1
altair                                  5.2.0
amqp                                    5.2.0
annotated-types                         0.6.0
anthropic                               0.16.0
antlr4-python3-runtime                  4.8
anyio                                   4.3.0
asgiref                                 3.8.1
asttokens                               2.4.1
async-timeout                           4.0.3
attrs                                   23.2.0
auto-labeling-pipeline                  0.1.23
backcall                                0.2.0
backoff                                 2.2.1
backports.zoneinfo                      0.2.1
beautifulsoup4                          4.12.3
billiard                                4.2.0
blinker                                 1.7.0
boto3                                   1.34.55
botocore                                1.34.55
bs4                                     0.0.2
BTrees                                  5.2
cachetools                              5.3.3
categorical-distance                    1.9
celery                                  5.3.6
certifi                                 2024.2.2
cffi                                    1.16.0
chardet                                 4.0.0
charset-normalizer                      3.3.2
click                                   8.1.7
click-didyoumean                        0.3.1
click-plugins                           1.1.1
click-repl                              0.3.0
cohere                                  4.51
colorama                                0.4.6
configparser                            6.0.1
cryptography                            42.0.5
cycler                                  0.12.1
dashscope                               1.14.1
dataclasses-json                        0.6.4
datasets                                2.13.2
datetime-distance                       0.1.3
decorator                               5.1.1
dedupe                                  2.0.23
dedupe-Levenshtein-search               1.4.5
dedupe-variable-datetime                0.1.5
defusedxml                              0.7.1
Deprecated                              1.2.14
dill                                    0.3.6
dirtyjson                               1.0.8
distro                                  1.9.0
dj-database-url                         0.5.0
dj-rest-auth                            2.2.8
Django                                  4.2.11
django-allauth                          0.52.0
django-celery-results                   2.4.0
django-cleanup                          6.0.0
django-cors-headers                     3.14.0
django-drf-filepond                     0.5.0
django-filter                           22.1
django-health-check                     3.18.1
django-polymorphic                      3.1.0
django-rest-polymorphic                 0.1.10
django-storages                         1.14.2
djangorestframework                     3.15.1
djangorestframework-xml                 2.0.0
doccano                                 1.8.4
docker-pycreds                          0.4.0
DoubleMetaphone                         1.1
drf-yasg                                1.21.7
easyinstruct                            0.1.2
environs                                9.5.0
et-xmlfile                              1.1.0
exceptiongroup                          1.2.0
executing                               2.0.1
fastavro                                1.9.4
filelock                                3.13.1
filetype                                1.2.0
flower                                  1.2.0
frozendict                              2.4.0
frozenlist                              1.4.1
fsspec                                  2024.2.0
furl                                    2.1.3
future                                  1.0.0
gitdb                                   4.0.11
GitPython                               3.1.42
google-api-core                         2.18.0
google-auth                             2.29.0
google-cloud-core                       2.4.1
google-cloud-storage                    2.16.0
google-crc32c                           1.5.0
google-resumable-media                  2.7.0
googleapis-common-protos                1.63.0
greenlet                                3.0.3
gunicorn                                20.1.0
h11                                     0.14.0
hanlp                                   2.1.0b56
hanlp_common                            0.0.19
hanlp_downloader                        0.0.25
hanlp_trie                              0.0.5
haversine                               2.8.1
highered                                0.2.1
httpcore                                1.0.4
httpx                                   0.27.0
huggingface-hub                         0.21.3
humanize                                4.9.0
hydra-core                              1.0.6
idna                                    3.6
importlib-metadata                      6.11.0
importlib_resources                     6.1.2
inflection                              0.5.1
ipdb                                    0.13.11
ipython                                 8.12.3
jedi                                    0.19.1
jiagu                                   0.2.3
jieba                                   0.42.1
Jinja2                                  3.1.2
jmespath                                1.0.1
joblib                                  1.3.2
jsonpatch                               1.33
jsonpointer                             2.4
jsonschema                              4.21.1
jsonschema-specifications               2023.12.1
kiwisolver                              1.4.5
kombu                                   5.3.6
langchain                               0.1.11
langchain-community                     0.0.27
langchain-core                          0.1.30
langchain-text-splitters                0.0.1
langsmith                               0.1.23
lexicalrichness                         0.5.1
llama-index                             0.10.22
llama-index-agent-openai                0.1.6
llama-index-cli                         0.1.11
llama-index-core                        0.10.22
llama-index-embeddings-huggingface      0.1.4
llama-index-embeddings-openai           0.1.7
llama-index-indices-managed-llama-cloud 0.1.4
llama-index-legacy                      0.9.48
llama-index-llms-openai                 0.1.12
llama-index-multi-modal-llms-openai     0.1.4
llama-index-program-openai              0.1.4
llama-index-question-gen-openai         0.1.3
llama-index-readers-file                0.1.12
llama-index-readers-llama-parse         0.1.3
llama-parse                             0.3.9
llamaindex-py-client                    0.1.13
lml                                     0.1.0
lxml                                    4.9.2
markdown-it-py                          3.0.0
MarkupSafe                              2.1.5
marshmallow                             3.21.1
matplotlib                              3.4.1
matplotlib-inline                       0.1.6
mdurl                                   0.1.2
mpmath                                  1.3.0
multidict                               6.0.5
multiprocess                            0.70.14
mypy-extensions                         1.0.0
neo4j                                   5.18.0
nest-asyncio                            1.6.0
networkx                                3.1
nltk                                    3.8.1
numpy                                   1.21.0
oauthlib                                3.2.2
omegaconf                               2.0.6
openai                                  1.9.0
openpyxl                                3.1.2
opt-einsum                              3.3.0
orderedmultidict                        1.0.1
orjson                                  3.9.15
packaging                               23.2
pandarallel                             1.6.5
pandas                                  1.5.3
parso                                   0.8.3
pathtools                               0.1.2
persistent                              5.2
phrasetree                              0.0.8
pickleshare                             0.7.5
Pillow                                  9.5.0
pip                                     24.0
pkgutil_resolve_name                    1.3.10
prometheus_client                       0.20.0
promise                                 2.3
prompt-toolkit                          3.0.43
proto-plus                              1.23.0
protobuf                                3.19.0
psutil                                  5.9.8
pure-eval                               0.2.2
pyarrow                                 15.0.0
pyasn1                                  0.6.0
pyasn1_modules                          0.4.0
pycparser                               2.21
pydantic                                2.6.4
pydantic_core                           2.16.3
pydeck                                  0.8.1b0
pyexcel                                 0.7.0
pyexcel-io                              0.6.6
pyexcel-xlsx                            0.6.0
Pygments                                2.17.2
pyhacrf-datamade                        0.2.7
pyhocon                                 0.3.60
PyJWT                                   2.8.0
PyLBFGS                                 0.2.0.15
PyLD                                    2.0.3
pyltp                                   0.4.0
Pympler                                 1.0.1
PyMuPDF                                 1.24.0
PyMuPDFb                                1.24.0
pynvml                                  11.5.0
pyparsing                               3.1.1
pypdf                                   4.1.0
python-dateutil                         2.9.0.post0
python-dotenv                           1.0.1
python3-openid                          3.2.0
pytorch-crf                             0.7.2
pytorch-transformers                    1.2.0
pytz                                    2024.1
pytz-deprecation-shim                   0.1.0.post0
PyYAML                                  6.0.1
referencing                             0.34.0
regex                                   2023.12.25
requests                                2.31.0
requests-oauthlib                       2.0.0
rich                                    13.7.1
rouge_score                             0.1.2
rpds-py                                 0.18.0
rsa                                     4.9
s3transfer                              0.10.0
sacremoses                              0.1.1
safetensors                             0.4.2
scikit-learn                            0.24.1
scipy                                   1.10.1
sentencepiece                           0.2.0
sentry-sdk                              1.40.6
seqeval                                 1.2.2
setuptools                              69.1.1
shortuuid                               1.0.12
simplecosine                            1.2
six                                     1.16.0
smmap                                   5.0.1
sniffio                                 1.3.1
soupsieve                               2.5
SQLAlchemy                              1.4.52
sqlitedict                              2.1.0
sqlparse                                0.4.4
stack-data                              0.6.3
streamlit                               1.24.0
striprtf                                0.0.26
subprocess32                            3.5.4
sympy                                   1.12
tenacity                                8.2.3
tensorboard                             2.4.1
tensorboardX                            2.5.1
termcolor                               2.4.0
textblob                                0.18.0.post0
texttable                               1.7.0
threadpoolctl                           3.3.0
tiktoken                                0.6.0
tokenizers                              0.15.2
toml                                    0.10.2
tomli                                   2.0.1
toolz                                   0.12.1
toposort                                1.5
torch                                   2.2.2+cu118
torchaudio                              2.2.2+cu118
torchvision                             0.17.2+cu118
tornado                                 6.4
tqdm                                    4.66.1
traitlets                               5.14.1
transformers                            4.38.2
typing_extensions                       4.10.0
typing-inspect                          0.9.0
tzdata                                  2024.1
tzlocal                                 4.3.1
ujson                                   5.6.0
Unidecode                               0.4.16
uritemplate                             4.1.1
urllib3                                 1.25.11
validators                              0.23.2
vine                                    5.1.0
waitress                                2.1.2
zope.interface                          6.2

**Comment by DW934** [comment]
I have this question too.Do you solve it ?

**Comment by JuliaTsai555** [comment]
![螢幕擷取畫面 2024-07-10 102308](https://github.com/doccano/doccano/assets/174773582/3ede64c3-b2f4-4ba4-8a58-b23eb91cecc8)
when you create a project, you should choose Sequence Labeling
